Kite SDK/PSPrintSDK/OLKiteViewController.m

```diff
@@ -133,63 +134,46 @@ - (void)transitionToNextScreen{
     
     // The screen we transition to will depend on what products are available based on the developers filter preferences.
     NSArray *groups = [OLProductGroup groupsWithFilters:self.filterProducts];
+    
+    UIStoryboard *sb = [UIStoryboard storyboardWithName:@"OLKiteStoryboard" bundle:nil];
+    NSString *nextVcNavIdentifier;
+    OLProduct *product;
     if (groups.count == 0) {
-        UIAlertView *av = [[UIAlertView alloc] initWithTitle:@"Store Maintenance" message:NSLocalizedString(@"Our store is currently undergoing maintence so no products are available for purchase at this time. Please try again a little later.", @"") delegate:self cancelButtonTitle:NSLocalizedString(@"OK", @"")  otherButtonTitles:nil];
-        av.tag = kTagNoProductsAlertView;
-        av.delegate = self;
-        [av show];
+        if ([UIAlertController class]){
+            UIAlertController *ac = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"Store Maintenance", @"") message:NSLocalizedString(@"Our store is currently undergoing maintence so no products are available for purchase at this time. Please try again a little later.", @"") preferredStyle:UIAlertControllerStyleAlert];
```

Comment:
AlertView on iOS 8 doesn't dismiss the OLKiteViewController in the same manner as iOS 7. Rather when you click on the OK button the alert disappears but the spinner is still showing. You probably want to dismiss. As an aside I'd probably refactor this into some helper that takes a block and does the right thing on iOS 7 & 8 otherwise I think testing gets more complicated as I really do need to go about trying both iOS 7 flow & 8 flow every time.